packages/datagateway-dataview/src/page/pageRouting.component.tsx

```diff
@@ -219,6 +227,42 @@ const SafeDLSDatasetsCardView = React.memo(
 );
 SafeDLSDatasetsCardView.displayName = 'SafeDLSDatasetsCardView';
 
+interface HomePageStateProps {
+  pluginHostUrl: string | undefined;
+}
+
+const SafeHomePage = React.memo(
```

Comment:
This probably makes sense more to be named "Translated" HomePage rather than "Safe".